src/content/docs/tutorial-create-alerts/improve-with-alerts.mdx

```diff
@@ -0,0 +1,94 @@
+
+<img
+    title="AQM flow"
+    alt="A diagram displaying how to use AQM to improve your system"
+    src={AQMflow}
+/>
+
+To get started with AQM, you need to do the following:
+
+<CollapserGroup>
+    <Collapser
+        id="AQM-dash"
+        title="1. Install the AQM dashboard"
+    >
+1. [Go to the **Alert Quality Management** instant observability page](https://newrelic.com/instant-observability/alert-quality-management)
+2. Click on **Install now**
+3. Choose an account to install the dashboard into
+4. View your dashboard
+    </Collapser>
+    <Collapser
+        id="KPIs"
+        title="2. Analyze your KPIs"
+    >
+With the dashboard installed, you can analyze key performance indicators, also known as KPIs. These KPIs are:
+
+* Incident count (alerts with a high number of incidents)
+* Accumulated incident time (alerts with high cumulative durations)
+* Mean time to close (the amount of time it takes until incidents are closed)
+* Percent under 5 minutes (the amount of incidents open for less than 5 minutes)
+
+The **Alerting Count by Policy** pane in the dashboard helps you identify these alert policies and determine any relevant patterns.
+    </Collapser>
+    <Collapser
+        id="first-enablement"
+        title="3. Begin the first AQM orientation"
+    >
+The AQM dashboard gives you a baseline of KPIs that you can use to begin the improvement process. You and your teams can review the most active policies from the previous step to reduce alert noise. Ask yourself questions about what the data is telling you and how you can fix them, such as:
+1. Are the alerts telling us something about a resource that needs to be fixed? If so, then fix the problem and see if the alert volume decreases.
+2. Are the alerts telling us about something that actually requires an immediate response? If not, then adjust or disable the policy.
+3. Are the policy thresholds set properly? If not, then consider adjusting the thresholds.
+
+Your response teams should deal with the incident alerts following these guidelines during and after the AQM orientation:
+
+* If you look at an alert and decide to take any sort of further investigative action, acknowledge the alert.
+* If you typically close an alert without doing anything else, do not acknowledge the alert.
+* If the incident alert is always on, do not close or acknowledge it. 
+    </Collapser>
+    <Collapser
+        id="gather-data"
+        title="4. Gather your data"
+    >
+It takes some time to accumulate your alert data from the dashboard. You should wait at least 2 weeks to gather this data, and regulary check to ensure that the incident responders for your alerts are following the guidelines outlined in the previous step.
```

Comment:
`2` s/be `two` https://learn.microsoft.com/en-us/style-guide/numbers

Reply:
Fixed!